languages/nix/injections.scm

```diff
@@ -1,39 +1,190 @@
-; mark arbitary languages with a comment
-((((comment) @injection.language) .
-  (indented_string_expression (string_fragment) @injection.content))
-  (#set! injection.combined))
-
-((binding
-   attrpath: (attrpath (identifier) @_path)
-   expression: (indented_string_expression
-     (string_fragment) @injection.content))
- (#match? @_path "(^\\w*Phase|(pre|post)\\w*|(.*\\.)?\\w*([sS]cript|[hH]ook)|(.*\\.)?startup)$")
- (#set! injection.language "bash")
- (#set! injection.combined))
+; ((comment) @content
+;   (#set! language "comment"))
 
-((apply_expression
-   function: (apply_expression function: (_) @_func)
-   argument: (indented_string_expression (string_fragment) @injection.content))
- (#match? @_func "(^|\\.)writeShellScript(Bin)?$")
- (#set! injection.language "bash")
- (#set! injection.combined))
+; ((comment) @language
+;   . ; this is to make sure only adjacent comments are accounted for the injections
+;   [
+;     (string_expression
+;       (string_fragment) @content)
+;     (indented_string_expression
+;       (string_fragment) @content)
+;   ]
+;   (#gsub! @language "/%*%s*([%w%p]+)%s*%*/" "%1")
+;   (#set! combined))
+
+; ; #-style Comments
+; ((comment) @language
+;   . ; this is to make sure only adjacent comments are accounted for the injections
+;   [
+;     (string_expression
+;       (string_fragment) @content)
+;     (indented_string_expression
+;       (string_fragment) @content)
+;   ]
+;   (#gsub! @language "#%s*([%w%p]+)%s*" "%1")
```

Comment:
Is this intentionally commented? It is not commented [upstream](https://github.com/nvim-treesitter/nvim-treesitter/blob/da89532f036da9855f3105955696df36f583b919/queries/nix/injections.scm) and has broken comment injecting language for strings:

<img width="381" alt="Screenshot 2025-02-10 at 00 50 44" src="https://github.com/user-attachments/assets/628058cc-ffba-4fb1-b2e8-89b8556e4e09" />